starknet_programs/get_number_l1_handler.cairo

```diff
@@ -0,0 +1,6 @@
+%lang starknet
+
+@l1_handler
+func get_number(from_address: felt) {
+    return ();
+}
```

Comment:
Maybe we could make this function return a number and then check it in the test

Reply:
l1 handlers can't return anything, I tried it but the compiler didn't allow it.